webapp/store/search.js

```diff
@@ -46,8 +46,8 @@ export const actions = {
     await this.app.apolloProvider.defaultClient
       .query({
         query: gql`
-          query findPosts($query: String!) {
-            findPosts(query: $query, limit: 10) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-27T13:49:08Z" title="Tuesday, August 27th 2019, 3:49:08 pm +02:00">Aug 27, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -46,8 +46,8 @@ export const actions = {
     await this.app.apolloProvider.defaultClient
       .query({
         query: gql`
-          query findPosts($query: String!) {
-            findPosts(query: $query, limit: 10) {
```


I find `query` a better naming for a string that is used for full-text search.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-27T14:08:52Z" title="Tuesday, August 27th 2019, 4:08:52 pm +02:00">Aug 27, 2019</time>_

---


I could agree with that, but if it's named query it breaks the functionality. not sure why exactly cause I'm not sure I completely understand the flow here, but it seems it passes through the `schema.gql` where we use the magic here https://github.com/Human-Connection/Human-Connection/blob/master/backend/src/schema/types/schema.gql#L10
then it gets passed to the `posts` resolver, which calls the `filterForBlockedUsers`
then cause there are no `badIds` it returns the `params`, otherwise it adds them to the `filter`, but the posts which match the query have already been returned from the database :point_up: